packages/@sanity/default-layout/package.json

```diff
@@ -6,9 +6,7 @@
   "scripts": {
     "coverage": "nyc --report=lcov --report=text _mocha -- test/**/*.test.js",
     "clean": "rimraf lib .nyc_output coverage",
-    "posttest": "npm run type-check",
-    "type-check": "tsc --noEmit -p tsconfig.dist.json",
-    "type-check:src": "tsc --noEmit"
+    "posttest": "npm run '"
```

Comment:
Is `npm run '` (apostrophe) a thing?

Reply:
Definitely not! Not sure what happened there 😳. Fixing.